Product/SoapUI_Test/RegressionSuite/Passthru/AuditLogging-Passthrough/AuditLogging-Passthrough-soapui-project.xml

```diff
@@ -18749,7 +18749,7 @@ if (propertiesFile.exists()) {
     </con:property>
     <con:property>
       <con:name>GatewayPropDir</con:name>
-      <con:value/>
+      <con:value>C:\app\wildfly-8.2.1.Final\wildfly-8.2.1.Final\modules\system\layers\base\org\connectopensource\configuration\main</con:value>
```

Comment:
Please remove this line.  Add this key-value pair to the SoapUI global properties so you never have to set this again.  Ping me if you have any questions on how to do this.